docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:focal-20210827
+FROM ubuntu:latest
```

Comment:
uhm, using `latest` sometimes may lead to some unpleasant surprises; if you do not have strong incentive, I would either leave the previous 'sealed' one or choose newer, but still, 'sealed'

Reply:
Changed.